enhancements/oc/inspect.md

```diff
@@ -60,8 +60,34 @@ truly generic, it has a generic fallback, but it understands many resources so t
 
 There are several special cases today.
 
-1. clusteroperators 
-   1. get all config.openshift.io resources
+1. any resource containing a `.status.relatedObjects` that has a schema conformant to
+
+   ```go
+    type <Foo>Status struct {
+     ... other fields
+	     	// relatedObjects is a list of objects that are "interesting" or related to this operator.  Common uses are:
+    	// 1. the detailed resource driving the operator
+    	// 2. operator namespaces
+    	// 3. operand namespaces
+    	// +optional
+    	RelatedObjects []ObjectReference `json:"relatedObjects,omitempty"`
```

Comment:
Will this also include "children" objects which already have owner references set?

Reply:
> Will this also include "children" objects which already have owner references set? Like not quite operands but more like objects created as a result of acting on the operand.

This does not cover it and we do not do that.  That is a good idea.